functions/go/set-standard-labels/transformer/const.go

```diff
@@ -0,0 +1,26 @@
+package transformer
+
+
+// https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/
+const (
+	AppLabelPrefix = "app.kubernetes.io/"
+	AppName = AppLabelPrefix + "name"
+	AppInstance = AppLabelPrefix + "instance"
+
+	// TBD
+	AppVersion = AppLabelPrefix + "version"
+	AppComponent = AppLabelPrefix + "component"
+	AppPartOf = AppLabelPrefix + "part-of"
+	AppManagedBy = AppLabelPrefix + "managed-by"
+)
```

Comment:
May be we can call the function `set-app-labels` since it's too specific to setting application labels.